app/src/main/java/de/westnordost/streetcomplete/quests/atm_cashin/AddAtmCashin.kt

```diff
@@ -0,0 +1,36 @@
+package de.westnordost.streetcomplete.quests.atm_cashin
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.CITIZEN
+import de.westnordost.streetcomplete.osm.Tags
+import de.westnordost.streetcomplete.quests.YesNoQuestForm
+import de.westnordost.streetcomplete.util.ktx.toYesNo
+
+class AddAtmCashin : OsmFilterQuestType<Boolean>() {
+
+    override val elementFilter = """
+        nodes with
+         amenity = atm
+         and !cash_in
+    """
```

Comment:
```suggestion
    override val elementFilter = "nodes with amenity = atm and !cash_in"
```
I think this fits quite well into one line 🤷